.github/workflows/release.yml

```diff
@@ -86,10 +86,7 @@ jobs:
           script: |
             export PATH="$(nix build github:runtimeverification/kup --no-link --json | jq -r '.[].outputs | to_entries[].value')/bin:$PATH"
             kup publish k-framework-binary .#k --keep-days 180
-            kup publish k-framework-binary .#k.openssl --keep-days 180
-            kup publish k-framework-binary .#k.procps --keep-days 180
-            kup publish k-framework-binary .#k.openssl.procps --keep-days 180
-            kup publish k-framework-binary .#k.procps.openssl --keep-days 180
```

Comment:
I don't think people really install these separately, so we only really need to cache the deps needed for blockchain plugin. perhaps we should even add an alias, something like `k.blockchain-plugin-deps`?

Reply:
Yeah, let's do that. It also avoid having to update instructions in lots of different places if we change the list of dependency libraries (as is happening here).